test/kitchen/kitchen-apt-check-azure.yml

```diff
@@ -16,6 +16,7 @@ transport:
 provisioner:
   product_name: chef
   product_version: 12.2.1
+  install_strategy: always
```

Comment:
🍰 

we appear to be using  `chef-solo` and `13.6.4` elsewhere, is this worth consolidating? (not sure why this is different).